frontend/src/component/PublicProfile/CustomFormManager.jsx

```diff
@@ -25,7 +25,7 @@ const DisplaySchemaPropertyString = props => {
       <span
         className={classnames(
           'DisplaySchemaPropertyString__label',
-          { noLabel: !props.label }
+          { noLabel: !props.label || props.label === ' ' }
```

Comment:
Can you add a comment explaining the string containing the space?

We may need to document this "feature" in the doc for writing json schemas too.

Reply:
Fixed